dsc/TOC.md

```diff
@@ -54,6 +54,10 @@
 ## [Using a DSC report server](reportServer.md)
 ## [Pull server best practices](secureServer.md)
 
+# [DSC examples](dscExamples.md)
+## [Building a CI-CD pipeline with DSC, Pester, and Visual Studio Team Services](dscCiCd.md)
```

Comment:
If you must use acronyms, then use CI/CD(as @eslesar mentioned), not CI-CD.
The first is widely used in the field, the later has no "meaning" :)